src/Common/nunit/Options.cs

```diff
@@ -349,17 +353,25 @@ public string[] GetValueSeparators ()
         protected static T Parse<T> (string value, OptionContext c)
         {
             Type tt = typeof (T);
+#if NETCORE
+            bool nullable = tt.GetTypeInfo().IsValueType && tt.GetTypeInfo().IsGenericType && 
+                !tt.GetTypeInfo().IsGenericTypeDefinition && 
+                tt.GetGenericTypeDefinition () == typeof (Nullable<>);
+            Type targetType = nullable ? tt.GetGenericArguments () [0] : typeof (T);
+#else
```

Comment:
I see you had to do this to compile. Since we don't do actually use any Nullable  types in our options, it makes me think we should eventually just build our own version of Mono.Options that has what we need and no more.